cristin-project/src/test/java/no/unit/nva/cristin/projects/query/QueryCristinProjectHandlerTest.java

```diff
@@ -689,4 +689,44 @@ private static Stream<? extends Arguments> queryResponseWithFundingFileReader()
         return Stream.of(Arguments.of(API_QUERY_RESPONSE_WITH_FUNDING_JSON));
     }
 
+
+    @Test
+    void shouldAddParamsToCristinQueryForFilteringAndReturnOk() throws IOException, ApiGatewayException {
+        cristinApiClientStub = Mockito.spy(cristinApiClientStub);
+        handler = new QueryCristinProjectHandler(cristinApiClientStub, new Environment());
+        var queryParams = Map.of("query", "hello",
+                "funding", "NRE",
+                "biobank", "123321",
+                "keyword", "nature",
+                "results", "5",
+                "unit", "184.12.60.0",
+                "sort", "start_date");
+        var input = requestWithQueryParameters(queryParams);
+        handler.handleRequest(input, output, context);
+        var captor = ArgumentCaptor.forClass(URI.class);
+
+        Mockito.verify(cristinApiClientStub).fetchQueryResults(captor.capture());
+        assertThat(captor.getValue().toString(),
```

Comment:
consider extracting `captor.getValue().toString()` into a local variable so we dont have to type the whole chain each time